lib/sqlalchemy/dialects/postgresql/asyncpg.py

```diff
@@ -47,6 +47,32 @@
     ``json_deserializer`` when creating the engine with
     :func:`create_engine` or :func:`create_async_engine`.
 
+.. _asyncpg_multihost_connecting:
+
+Multihost Connections
```

Comment:
**mike bayer** (zzzeek) wrote:

have this be only a link to the existing psycopg section. formats are identical

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4755

Reply:
I made a commit, but I'm not sure this is it. Can you check or maybe show an example?

Reply:
**mike bayer** (zzzeek) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4755